v3/qvex/lynepad2/lynepad2.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "QVEX Lynepad",
+    "productId": "0xE678",
+    "vendorId": "0x04D8",
+    "matrix": 
+    {
+        "rows": 5,
+        "cols": 4
+    },
+    "layouts":
+    {
+        "keymap":
+        [
+            ["0,0","0,1","0,2","0,3"],
+            ["1,0","1,1","1,2","1,3"],
+            ["2,0","2,1","2,2"],
+            [{"y":-0.75,"x":4.5},"3,0"],
+            [{"x":3.5},"3,2","4,0\n\n\n\n\n\n\n\n\ne0","3,3"],
+            [{"x":4.5},"3,1"],
+            [{"y":0.5,"x":3.25},"4,1\n\n\n\n\n\n\n\n\ne1"]
+        ]
+    },
+    "menus": ["qmk_backlight","qmk_rgblight"],
```

Comment:
I believe you want this instead

`[qmk_backlight_rgblight](https://www.caniusevia.com/docs/built_in_menus#qmk_backlight_rgblight)`

https://www.caniusevia.com/docs/custom_ui

Reply:
Thanks for pointing this out. I ended up creating a custom menu that is more suitable for what the Lynepad can do.